qutebrowser/keyinput/keyutils.py

```diff
@@ -298,6 +299,9 @@ def __str__(self):
             elif self.modifiers == Qt.NoModifier:
                 assert not is_special(self.key, self.modifiers)
                 return key_string.lower()
+            elif self.modifiers == Qt.KeypadModifier:
+                assert not is_special(self.key, self.modifiers)
+                return key_string.lower()
```

Comment:
Doing this means `Num+1` will only be printed as `1`, which is a "lie" and will make debugging things very difficult (as they're still different keybindings).

Instead, I'd prefer keeping the special keybinding syntax `<Num+1>`, which should work if you just adjust the assertion below (or place it under an `if self.modifiers != Qt.KeypadModifier:`).